app/src/main/java/com/piwniczna/mojakancelaria/activities/add_client/AddClientFragment.kt

```diff
@@ -40,7 +39,7 @@ class AddClientFragment : Fragment() {
     }
 
     fun handleSaveClient(view: View) {
-        val newClientName = newClientEditText.text.toString()
+        val newClientName = new_client_edit_text.text.toString()
```

Comment:
```suggestion
        val newClientName = clientEditText.text.toString()
```
it should be name of EditText variable, not EditText id